### ISSUE TITLE
Compile error on Raspberry Pi OS 32bit

### ISSUE THREAD

**Issue by dlsniper** [hybridgroup/gocv#761]
## Description
```
pi@raspberrypi:~/go/src/gocv.io/x/gocv $ go run ./cmd/version/main.go 
# gocv.io/x/gocv
./core.go:2009:16: type [1073741824]*_Ctype_char larger than address space
./core.go:2009:16: type [1073741824]*_Ctype_char too large
```

The line goes like this:
```
tmpslice := (*[1 << 30]*C.char)(unsafe.Pointer(strs.strs))[:length:length]
```
And the fix is trivial,
```
tmpslice := (*[1 << 28]*C.char)(unsafe.Pointer(strs.strs))[:length:length]
```
As far as I can tell, it doesn't cause any problems, but I tested this in a limited capacity. Should I send a PR to address this?

## Your Environment

* Operating System and version:
Raspberry Pi OS
Version: August 2020
Release date: 2020-08-20
* OpenCV version used:
gocv version: 0.25.0
opencv lib version: 4.5.0
* How did you install OpenCV?
I ran the `make install` script, per website instructions.
* GoCV version used:
0.25.0
* Go version:
go version go1.15.3 linux/arm
* Did you run the `env.sh` or `env.cmd` script before trying to `go run` or `go build`?
No
* Hardware platform
Raspberry Pi 3

**Comment by deadprogram** [comment]
Hi @dlsniper you can probably even reduce to `1 << 20` in most cases. Yes, we'd appreciate a PR.

**Comment by dlsniper** [comment]
For the PR, what would be the preferred solution? Just replace it with `1<<20` everywhere? Or something configurable? I did not have the time to study the code yet to be confident in a solution without some advice first. Thank you!

**Comment by deadprogram** [comment]
Previous PRs have used `1 << 27`: 

https://github.com/hybridgroup/gocv/issues/90

I think we can safely change that everywhere. The CI build will let us know if that is true, so low risk.

**Comment by freman** [comment]
I have tested 1 << 27 on my pi 4 and it works, once they release the 64 bit of the raspberry pi os (or whatever it's called now) should be a moot point yeh?

**Comment by dlsniper** [comment]
> once they release the 64 bit of the raspberry pi os (or whatever it's called now) should be a moot point yeh?

Unless you are still using a 32bit bit OS because you don't need anything specific to the 64bit arch.

**Comment by deadprogram** [comment]
This was released with v0.26.0 so now closing. Thank you everyone!
